rw-system.sh

```diff
@@ -198,11 +198,27 @@ if [ "$(getprop ro.vendor.product.manufacturer)" = "motorola" ] || [ "$(getprop
     fi
 fi
 
-if getprop ro.vendor.build.fingerprint | grep -q -i -e xiaomi/wayne -e xiaomi/jasmine; then
+if getprop ro.vendor.build.fingerprint | grep -q -i -e xiaomi/wayne -e xiaomi/jasmine -e xiaomi/jasmine_sprout; then
     setprop persist.imx376_sunny.low.lux 310
     setprop persist.imx376_sunny.light.lux 280
     setprop persist.imx376_ofilm.low.lux 310
     setprop persist.imx376_ofilm.light.lux 280
+    setprop ro.sf.lcd_density 460
+
+    # VoLTE
```

Comment:
What is the issue you're trying to fix?
What's the behaviour with/without ims.apk without this prop, and with this prop?

Reply:
After looking, VoLTE works without this tweak, and so does the Dual Camera. Don't merge that last block, except for the lcd_density.